steps/draft-appeals/DraftAppeals.js

```diff
@@ -25,7 +29,16 @@ class DraftAppeals extends RestoreAllDraftsState {
 
   get drafts() {
     const draftCases = this.req.session.drafts;
-    return draftCases;
+    const isIba = benefitTypeUtils.isIba(this.req);
+    return Object.fromEntries(
+      // eslint-disable-next-line no-unused-vars
+      Object.entries(draftCases).filter(([key, caseData]) => {
+        const { benefitType } = caseData.BenefitType || '';
```

Comment:
destructuring `''` into `{ benefitType }` is slightly odd. Maybe `{}` makes more sense?